woocommerce/woocommerce-android
# Switch to upload key for release bundle signing

This PR introduces signing with upload key. The upload key is used to sign the _release_ bundle that is uploaded to the Play Store. The local _release_ universal apk is still signed with the master key in order to test with the production configuration. 

To Test:
1. Checkout a new branch from this one (name should be `release/<something>`).
2. Bump the `versionCode` in `build.gradle` and commit.
3. Pull the latest `mobile-secrets` repository.
4. Run `./gradlew installDeps` to update the local configuration.
5. Run `bundle exec fastlane build_and_upload_beta`.
6. Verify that the new binary is uploaded to the Play Store Artifact library.
7. Verify that the universal apk in your local `artifacts` folder can be installed on a device on top of a previous version and that Google Login works. 


Update release notes:
